stable/datadog/templates/agent-clusterchecks-pdb.yaml

```
@@ -0,0 +1,17 @@
+{{- if .Values.clusterChecksRunner.createPodDisruptionBudget -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "datadog.fullname" . }}-clusterchecks
+  labels:
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app.kubernetes.io/name: "{{ template "datadog.fullname" . }}"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+spec:
+  minAvailable: 1
```

sorry I didn't mention which configuration make sens for the `clusterchecks`.

Could you use `maxUnavailable` instead of `minAvailable`

```suggestion
  maxUnavailable: 1
```

---

done. according to [the docs](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget) these two fields are mutually exclusive so i just went with maxUnavailable 👍